zazu/repo/commands.py

```diff
@@ -51,10 +55,22 @@ def ci(ctx):
 
 
 @repo.command()
+@click.argument('repository')
+@click.option('--nohooks', is_flag=True)
 @click.pass_context
-def clone(ctx):
+def clone(ctx, repository, nohooks):
     """Clone and initialize a repo"""
-    raise NotImplementedError
+    try:
+        Git().clone(repository)
```

Comment:
should we init submodules by default recursively? Probably...maybe add an option?